packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -188,9 +188,15 @@ const VisitHeader: React.FC = () => {
           )}
           <ConfigurableLink className={styles.navLogo} to="${openmrsSpaBase}/home">
             <div className={styles.divider}>
-              <svg role="img" width={110} height={40}>
-                <use xlinkHref="#omrs-logo-white"></use>
-              </svg>
+              {logo?.src ? (
+                <img className={styles.logo} src={logo.src} alt={logo.alt} width={110} height={40} />
+              ) : logo?.name ? (
+                logo.name
+              ) : (
+                <svg role="img" width={110} height={40}>
+                  <use xlinkHref="#omrs-logo-white"></use>
+                </svg>
+              )}
```

Comment:
```suggestion
const Logo = ({ logo }) => {
  if (logo?.src) {
    return (
      <img
        className={styles.logo}
        src={logo.src}
        alt={logo.alt}
        width={110}
        height={40}
      />
    );
  }

  if (logo?.name) {
    return <span>{logo.name}</span>;
  }

  return (
    <svg role="img" width={110} height={40}>
      <use xlinkHref="#omrs-logo-white"></use>
    </svg>
  );
};
```

And then you could use 
```tsx
<div className={styles.divider}>{<Logo logo={logo} />}</div>
```